File: YoutubeLinks.Shared/Features/Playlists/Helpers/PlaylistHelpers.cs
namespace YoutubeLinks.Shared.Features.Playlists.Helpers
{
    public static class PlaylistHelpers
    {
        public static string PlaylistFileTypeToString(PlaylistFileType playlistFileType)
        {
            return playlistFileType switch
            {
                PlaylistFileType.Txt => "txt",
                _ => "json",
            };
        }
    }
}

File: youtube-links/src/shared/features/playlists/helpers/PlaylistHelpers.ts
import { PlaylistFileType } from './PlaylistFileType';

export namespace PlaylistHelpers {
	export function playlistFileTypeToString(playlistFileType: PlaylistFileType): string {
		switch (playlistFileType) {
			case PlaylistFileType.TXT:
				return 'txt';
			default:
				return 'json';
		}
	}
}
